core/src/main/java/com/graphhopper/routing/VirtualEdgeIterator.java

```diff
@@ -135,6 +132,12 @@ public EdgeIteratorState setName( String name )
     }
 
     @Override
+    public boolean getBoolean(int key, boolean _default, PMap kwargs)
```

Comment:
I thought the default construct is helpful, not to implement the key logic in every EdgeIteratorState but only where it is necessary (in our case in the VirtualEdgeIteratorState ). 
That is, if an Edge does not know how to handle a key, it just returns the default.

I think this behaviour is not really necessary, but  saved me from a lot of code duplications (```if key == DISPREFERED_EDGE return false```)